docs/src/main/sphinx/connector/redshift.md

```diff
@@ -64,6 +64,32 @@ documentation](https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-configura
 ```{include} jdbc-authentication.fragment
 ```
 
+### UNLOAD configuration
+
+This feature enables using Amazon S3 to efficiently transfer data out of Redshift
```

Comment:
Please describe what this actually does for user .. seems like it is only a performance improvement .. right? And if so .. why is it called unload?

Reply:
This feature uses different approach for performance improvements. Additionally, it requires additional configs to enable this feature. Please suggest text improvements.

Reply:
So lets move that into the performance section and explain more there.

Reply:
I assume we need to explain that they need a S3 account and whatever else .. and when this should and should not be used.. I am not aware of this info so I cant really reword appropriately without more details